bindings/c/include/opendal.h

```diff
@@ -147,6 +148,16 @@ typedef struct opendal_result_read {
   enum opendal_code code;
 } opendal_result_read;
 
+/*
+ The result type for [`opendal_operator_is_exist()`], the field `exist`
+ contains whether the path exists, and the field `code` contains the
+ corresponding error code.
+ */
+typedef struct opendal_result_exist {
```

Comment:
I tend to let the similar types have the same prefix, e.g. `opendal_result_`. The naming of `opendal_operator_is_exist_result` is a little bit complex in the first place and is kind of ambiguous since user might think it as `Whether the operator exists?`

Reply:
Great!

Reply:
By the way, is it a good idea for use diffent naming style for struct and function? For example:

- `struct`: `OpendalOperator`, `OpendalResultOperatorIsExist`
- `function`: `opendal_operator_is_exist`

Reply:
> By the way, is it a good idea for use diffent naming style for struct and function? For example:
> 
> * `struct`: `OpendalOperator`, `OpendalResultOperatorIsExist`
> * `function`: `opendal_operator_is_exist`

I think this is a good idea in most cases.
But AFAIK, in C, most libraries and programs use `snake_case` for structs, functions and variables, and in C++, they use CamelCase.

Reply:
> But AFAIK, in C, most libraries and programs use `snake_case` for structs, functions and variables,

Ok, let's keep the style As-IS.